<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionemploye</class>
 <widget class="QDialog" name="gestionemploye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1472</width>
    <height>1296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1531</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Employ√©e</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>10</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>   Recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:black;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;
</string>
     </property>
     <property name="text">
      <string>search</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>491</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="salaire_e_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="mail_e_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>250</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_e_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:blue ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;
</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_e_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_e_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>salaire</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>260</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mail :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>nom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_e_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>prenom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_e_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>270</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mail :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mail_e_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>260</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>salaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="salaire_e_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom_e_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>nom :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_e_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="UPDATE_3">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:blue ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;
</string>
       </property>
       <property name="text">
        <string>mettre a jour</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="stat_3">
      <attribute name="title">
       <string>stat</string>
      </attribute>
      <widget class="QWidget" name="stats_5" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>441</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mailing_3">
      <attribute name="title">
       <string>mailing</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>451</width>
         <height>381</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QTextEdit" name="msg_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>381</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'NSimSun'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="placeholderText">
         <string>Ton message ici ...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="mail_pass_3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>24</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>ton mot de passe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="file_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>94</y>
          <width>231</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>ajouter des fichiers</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rcpt_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>24</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
        </property>
        <property name="placeholderText">
         <string>destinataire</string>
        </property>
       </widget>
       <widget class="QPushButton" name="browseBtn_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>94</y>
          <width>84</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 68 ;
padding-left: 1px;
padding-right: 1px;
min-width: 80px;
max-width: 30px;
min-height: 8px;
max-height: 15px;
font: 75 10pt &quot;NSimSun&quot;;
}</string>
        </property>
        <property name="text">
         <string>parcourir</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="sujet_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>381</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
font: 75 12pt &quot;NSimSun&quot;;
}</string>
        </property>
        <property name="placeholderText">
         <string>sujet</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sendBtn_3">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>300</y>
          <width>182</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 68 ;
padding-left: 50px;
padding-right: 50px;
min-width: 80px;
max-width: 80px;
min-height: 8px;
max-height: 15px;
font: 75 10pt &quot;NSimSun&quot;;

}</string>
        </property>
        <property name="text">
         <string>envoyer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QTableView" name="tab_employe_5">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>80</y>
       <width>641</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QTableView {
     selection-background-color: qlineargradient(rgb(0, 170, 255));
font: 75 12pt &quot;NSimSun&quot;;
 }
 QTableView QTableCornerButton::section {
     background:  rgb(0, 0, 255);
     border: 2px outset  rgb(0, 0, 255);
 }</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>580</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/icons/logo.png);
</string>
     </property>
     <widget class="QLineEdit" name="id_e_s_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>-20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:blue ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;</string>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_76">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;NSimSun&quot;;</string>
     </property>
     <property name="text">
      <string>Trier par :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:red ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;
</string>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>480</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:blue ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;</string>
     </property>
     <property name="text">
      <string>generer pdf </string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer_e_3">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>480</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:red ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;
</string>
     </property>
     <property name="text">
      <string>Effacer</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>520</y>
       <width>381</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Authentification</string>
     </property>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>321</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt;&quot;&gt;Ajouter un utilisateur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idajout_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>84</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>84</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: rgb(245, 245, 245);
background-color: rgb(0, 0, 255);
border-width: 1px;
border-color: rgb(0, 0, 255);
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 68 ;
padding-left: 1px;
padding-right: 1px;
min-width: 80px;
max-width: 30px;
min-height: 8px;
max-height: 15px;
font: 75 10pt &quot;NSimSun&quot;;
}</string>
      </property>
      <property name="text">
       <string>S'inscrire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Entrez votre mpt de passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mpajout_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_41">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>155</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Entrez votre nom d'utilisateur</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tab_employe_6">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>540</y>
       <width>251</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QTableView {
     selection-background-color: qlineargradient(rgb(0, 170, 255));
font: 75 12pt &quot;NSimSun&quot;;
 }
 QTableView QTableCornerButton::section {
     background:  rgb(0, 0, 255);
     border: 2px outset  rgb(0, 0, 255);
 }</string>
     </property>
    </widget>
    <widget class="QWidget" name="stats_6" native="true">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>520</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
